src/lib/utils/fiat-estimates/fiat-estimates.ts

```diff
@@ -0,0 +1,202 @@
+import { derived, writable, get } from 'svelte/store';
+import assert from '$lib/utils/assert';
+import deduplicateReadable from '../deduplicate-readable';
+import tokensStore from '$lib/stores/tokens/tokens.store';
+import createSocket from './sockets';
+import { BinanceCommand, BinanceMessage } from './binance-ws-api/types';
+import { SUPPORTED_SYMBOLS, type SupportedSymbol } from './binance-ws-api/supported-symbols';
+import type { z } from 'zod';
+
+interface Amount {
```

Comment:
Oh ya. Let's do that in a separate PR then and for all `Amount` types, because this isn't the only place (and i'd rather have them all be the same so you can pass them around).